tox.ini

```diff
@@ -1,11 +1,17 @@
 [tox]
 envlist =
     lint
-    py{35,36,37}-django111
-    py{35,36,37,38}-django22
+    py{36,37}-django111
```

Comment:
Are we ready to drop support for Django 1.11 here as well, like we did in azavea/docker-django#113?

Reply:
I think so. We just need to remove it from `setup.cfg` too.